packages/login/theme/lumo/vaadin-login-form-styles.js

```diff
@@ -1,13 +0,0 @@
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
-
-registerStyles(
-  'vaadin-login-form',
```

Comment:
Note, `vaadin-login-form` does not have shadow DOM, so with Polymer this was simply rendered to light DOM. However, with LitElement based version this didn't work. So I decided to remove this file altohether.